css/style.css

```diff
@@ -0,0 +1,97 @@
+.flex-box {
+  display: flex;
+}
+
+.body {
+  background-color: #e5e5e5;
+}
+
+.container {
+  width: 325px;
+  justify-content: center;
+  border: 1px solid blue;
+}
```

Comment:
- [ ] Please remove the purple border placed in the container.
Here is the Figma Design you should replicate
![image](https://user-images.githubusercontent.com/50941074/158786258-7838dcdd-131f-4989-8527-3b7e5f64b92a.png)